new/client-rb-logger.md

```diff
@@ -0,0 +1,39 @@
+---
+RFC: unassigned
+Title: Logging for chient.rb
+Author: Zeal Jagannatha <zeal@fb.com>
+Status: Draft
+Type: Standards Track
+Tracking:
+  - https://github.com/chef/chef/issues/5526
+---
+
+# Logging for client.rb
+
+Output from client.rb doesn't get logged to the log file specified with --logfile.
+If you want to save the output from client.rb along-side the chef logs, you need to redirect stdout, rather than using chef's built-in logging.
+I'd like to make Chef::Log accessible from the client.rb so messages logged during client.rb execution can be stored using chef's built-in logging.
+
+## Motivation
+
+    As a chef user,
```

Comment:
This isn't your real motivation I don't think. I'm guessing you are trying to do some debugging on values loaded during the config file? Whatever the real underlying motivation is, you should document it here.